functions/vcs.zsh

```diff
@@ -110,7 +110,18 @@ function +vi-hg-bookmarks() {
 
 function +vi-vcs-detect-changes() {
   if [[ "${hook_com[vcs]}" == "git" ]]; then
-    vcs_visual_identifier='VCS_GIT_ICON'
+
+    local remote=`git ls-remote --get-url`
+    if [[ "$remote" =~ "github" ]] then
```

Comment:
Yep. I have one question also. If we would like to refactor that a bit, like extract the code in that condition (I don't like it so much now) how would you do? 

I just want the function `+vi-vcs-detect-changes()` to look simpler. I tried to put is separate function but never make it to work when returning the icon identifier, not so much bash under my belt.